frontend/src/views/ProjectView.vue

```diff
@@ -46,10 +47,19 @@ const {
 } = useProjectQuery(projectId);
 
 const {
-    data: group,
-    isLoading: isGroupLoading,
-    isError: isGroupError,
-} = useUserGroupQuery(projectId);
+    data: groups,
+    isLoading: isGroupsLoading,
+    isError: isGroupsError,
+} = useProjectGroupsQuery(projectId);
+
+const { data: user } = useCurrentUserQuery();
+
+const group = computed(() => {
+    if (!groups.value) return null;
+    return groups.value.filter((group) =>
+        group.members.some((member) => member.uid === user.value?.uid)
+    )[0];
+});
```

Comment:
Dit werkt niet als de gebruiker niet in een groep zit.

Waarschijnlijk was het de bedoeling om hier `useProjectGroupQuery` te gebruiken?